src/core/app/userapp/create_user.go

```diff
@@ -0,0 +1,61 @@
+package userapp
+
+import (
+	"context"
+
+	"github.com/ymdd1/mytweet/src/core/domain/userdm"
+	"github.com/ymdd1/mytweet/src/core/domain/vo"
+)
+
+type CreateUserApp struct {
+	userRepository userdm.UserRepository
+}
+
+func NewCreateUserApp(userRepo userdm.UserRepository) *CreateUserApp {
+	return &CreateUserApp{
+		userRepository: userRepo,
+	}
+}
+
+type CreateUserRequest struct {
+	Email      vo.Email
+	Password   vo.Password
+	Created_at vo.Created_at
+	Updated_at vo.Updated_at
```

Comment:
```suggestion
	CreatedAt vo.CreatedAt
	UpdatedAt vo.UpdatedAt
```

キャメルで書きましょう！

他もキャメルで書いていないところは修正してください。

Reply:
上記の通りに修正しました。